mifospay/src/main/res/layout/faq_list_child.xml

```diff
@@ -8,7 +8,10 @@
         android:id="@+id/faq_list_child"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_margin="16dp"
+        android:layout_marginLeft="40dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginBottom="16dp"
```

Comment:
Hey, @refactor-droidyy  your change looks great. But it would be much better if you avoid using hardcoded values, instead use the predefined values from `dimens.xml` file or add a new one if it is not there

Reply:
Ya, sure I will soon update this PR.

Reply:
Ya, sure I will soon update this PR.